app/views/notes/_notes.html.erb

```diff
@@ -34,7 +34,7 @@
 	    <%= distance_of_time_in_words(Time.at(node.latest.timestamp), Time.current, { include_seconds: false, scope: 'datetime.time_ago_in_words' }) %>
           <% end %>
           | <i class="fa fa-eye"></i> <%= number_with_delimiter(node.totalviews) %> <span class="hidden-xs hidden-sm"><%= t('notes._notes.views') %></span>
-          | <i style="<% if node.likes > 0 %>color:#db4;<% else %>color:#888;<% end %>" class="fa fa-star-o"></i> <%= node.likes %>
+          | <i style="<% if node.likers.length > 0 %>color:#db4;<% else %>color:#888;<% end %>" class="fa fa-star-o"></i> <%= node.likers.length %>
```

Comment:
we might do `likers.count` so that it runs a COUNT sql instead of calling the whole record and then counting it.

Reply:
@jywarren I made this update and committed just now, but then I remembered I had considered it at some point earlier and left it as `length`. So I double checked the rails logs and I think because plots2 implements cached likes for nodes, it doesn't make a difference if we use `length` or `count` for this specific view (its always the exact same number of queries with similar times on average). I was thinking for consistency with `_likes.html.erb` to leave it as `length`? 

I reverted it back to `length` for now but let me know your thoughts!